src/main/java/org/systemexception/lunarlander/model/Physics.java

```diff
@@ -0,0 +1,36 @@
+package org.systemexception.lunarlander.model;
+
+import java.math.BigDecimal;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+
+/**
+ * @author leo
+ * @date 28/12/15 21:30
+ */
+public class Physics {
+
+	private double v, s;
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Perhaps 's' could be replaced by a local variable.](https://www.codacy.com/app/leo_4/lunar-lander/file/1677825521/issues/source?bid=3040650&fileBranchId=3040754#l14)